operations_on_datastructures/array_left_rotation_efficient.cpp

```diff
@@ -0,0 +1,64 @@
+#include <iostream>
+/**
```

Comment:
This is good info but will not generate proper documentation by our auto tools. Please follow the style [doxygen][https://www.doxygen.nl/manual/docblocks.html) and see #807 for example.
Inside the comment blocks, you can use markdown syntaxes for better formatting. This will generate documentation like [this](https://kvedala.github.io/C-Plus-Plus/da/df2/durand__kerner__roots_8cpp.html). This example also shows you about how to write a test case